src/v0/sources/mailjet/transform.js

```diff
@@ -47,6 +48,9 @@ function process(event) {
     // If userId is not present, then generating it from email using md5 hash function
     message.userId = md5(event.email);
   }
+  if (message.type === "track") {
+    throw new CustomError("track is not supported", 400);
```

Comment:
still using this? if not urgent wait for other PR to merge and use new error types

Reply:
no no we will not merge it dileep. Testing the source transformation alerts. This will never be merged.